src/demo-app/a11y/menu/menu-a11y.html

```diff
@@ -0,0 +1,54 @@
+<div class="demo-menu">
+  
+  <section>
+    <h2>Click on the icon will trigger menu</h2>
+    <md-menu #menu1="mdMenu">
+      <button md-menu-item> Settings </button>
+      <button md-menu-item> Help </button>
```

Comment:
Can happen in a follow-up PR, but we got some advice that the buttons should _do_ something as well. Easiest thing is to just show a snackbar (which will read something w/ `aria-live`) like `You clicked "Settings"`